internal/scan-manager/core/pom.xml

```diff
@@ -168,11 +186,22 @@
                     <path>/</path>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven.surefire.plugin.version}</version>
```

Comment:
Please move surefire plugin version (`pluginManagement`) also to parent, so that every module will use the same version. 

https://github.com/wso2/security-tools/blob/master/internal/scan-manager/pom.xml